static/app/components/assigneeSelector.tsx

```diff
@@ -296,22 +296,46 @@ class AssigneeSelector extends React.Component<Props, State> {
   renderNewDropdownItems(): ItemsBeforeFilter {
     const teams = this.renderNewTeamNodes();
     const members = this.renderNewMemberNodes();
+    const suggestedAssignees = new Set(this.renderSuggestedAssigneeNodes() ?? []);
+    const assigneesSearchKey: string[] = [];
+    const filteredTeams: ItemsBeforeFilter = [];
+    const filteredMembers: ItemsBeforeFilter = [];
+
+    // filter out duplicates of Team/Member if also a Suggested Assignee
+    if ([...suggestedAssignees].length) {
+      [...suggestedAssignees].filter(assignee =>
+        assigneesSearchKey.push(assignee.searchKey)
```

Comment:
seems like we use the `searchKey` to check for duplicates, but it is possible (though slim chance) that a team and a member could share the same search key if a team's slug matched a user's email or name. 

Perhaps we should check a combination of assignee type (`team`, `member`) and assignee id. I know the suggestedOwners array in this components state has an `owner` field which follows the form `<type>:<id>` for example `member:2` or `team:5`. It might be easier to construct a set from `suggestedOwners` and then filter `teams` and `members` by that set. lmk if that makes sense or you need any clarification.